src/views/github/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from "react";
+import { Redirect } from 'react-router-dom';
+
+import { parseOAuthCode } from './../../actions';
+
+
+// TODO: use state for verification + dont redirect
+export const OAuthCallbackView = () => {
+  const [code, setCode] = useState<any>("")

```

Comment:
are we sure it's any? we should be avoiding any as much as possible

Reply:
changed to `string`